src/compiler/diagnosticMessages.json

```
@@ -1364,6 +1364,46 @@
         "category": "Error",
         "code": 1433
     },
+    "Unexpected keyword or identifier.": {
+        "category": "Error",
+        "code": 1434
+    },
+    "Unknown keyword or identifier. Did you mean '{0}'?": {
+        "category": "Error",
+        "code": 1435
+    },
+    "Decorators must precede all other keywords for property declarations.": {

```

It's not a huge deal, but consider

```suggestion
    "Decorators must precede the name and all keywords of property declarations.": {
```